### ISSUE TITLE
Question about reproducing Occ-ScanNet-mini results with mono_vggt_bin16_mini_release.py

### ISSUE THREAD

**Issue by Kybin-kb** [JuIvyy/GPOcc#5]
<img width="1165" height="376" alt="Image" src="https://github.com/user-attachments/assets/686798fa-b18c-477b-a6a7-b65472fbc6f6" />

<img width="1386" height="524" alt="Image" src="https://github.com/user-attachments/assets/5d1d0e7a-19fd-43b6-baaa-3548061b3bbf" />

Hi, thank you very much for open-sourcing this great work.

I trained the Occ-ScanNet-mini version using the config:

mono_vggt_bin16_mini_release.py

After training for 10 epochs, I got the following validation results:

Epoch 10 IoU of sem is about 0.480
Epoch 10 IoU of geo is about 0.540

However, compared with the results reported in the paper on the full Occ-ScanNet dataset, there seems to be a gap of nearly 10 points in IoU / mIoU.

I would like to ask:

Is this mainly because I only trained for 10 epochs, and more training epochs are required to reproduce the paper results?
Or is this gap expected because I am using the Occ-ScanNet-mini dataset instead of the full Occ-ScanNet dataset?
Could you please provide the expected IoU / mIoU range for the Occ-ScanNet-mini setting, or the recommended training schedule for this config?

Thank you again for your work and help!

**Comment by Kybin-kb** [comment]
I am trying to reproduce the monocular VGGT setting on Occ-ScanNet-mini. I noticed that the repository provides `mono_vggt_bin16_release.py` for the full Occ-ScanNet setting, but I could not find a separate mini config. I changed only the dataset field from `data_tg='base'` to `data_tg='mini'` for both train and val.

My current setting is:
- config: based on `mono_vggt_bin16_release.py`
- model: `VGGTGaussianSegmentor`
- `num_bins=16`
- `opacities_threshold=0.01`
- `frozen_backbone=False`
- `freeze_blocks=0`
- optimizer: AdamW, lr `2e-4`, weight decay `0.01`
- batch size: 2 per GPU
- `max_epochs=20`
- Occ-ScanNet-mini has about 688 iterations per epoch in my run

With this setting, the mini run plateaus around 48-49 mIoU after about 13 epochs. In contrast, my full Occ-ScanNet run reaches around 55 mIoU by epoch 6, which is close to the paper’s reported full-data result.

Could you please clarify the recommended training setup for Occ-ScanNet-mini with GPOcc-VGGT?

Specifically:
1. Did you use a separate mini config for `mono_vggt_bin16`?
2. What are the recommended `max_epochs` or total training iterations for Occ-ScanNet-mini?
3. What mIoU/IoU should we expect for GPOcc-VGGT on Occ-ScanNet-mini?

Any mini config, training command, or expected validation log would be very helpful.

Thank you!

**Comment by JuIvyy** [comment]
Sorry, as in the paper, we didn't conduct experiments on mini subset, thus we don't know what's the best hyperparameter. Please do not compare metrics between full and mini subset.

If you want to try small experiments for ablation, you can use depth-anything version.

**Comment by Kybin-kb** [comment]
thanks！
